src/analyzer/ipdialog.cpp

```diff
@@ -95,9 +94,9 @@ bool IpDialog::eventFilter(QObject *obj, QEvent *event)
 						break;
 
 					case Qt::Key_0:
-						if (edit->text().isEmpty() || edit->text() == "0")
+						if (edit->text().isEmpty() || edit->text().endsWith("0"))
```

Comment:
According to the change this would be a undesired change as what we'd rather do is change focus to next element if there is a "0" in the given QLineEdit.